### PR TITLE
DOC: Moved the ChatOps deployment steps and requirements to an INSTALL.md file

### PR DESCRIPTION
**Pull request by Neonbluestoplight**
The main changes made were:

* The deployment instructions and requirements were moved from README.md to INSTALL.md

* A few fixes were made for the local run instructions using docker. 

* A small `Dependencies` section was also added within `Deployment` to make sure users know of Python 3.10+ requirements (if running from source) and docker user group addition (if running via docker)

### CODE REVIEW

**Review by meoflynn — DISMISSED** [review]

File: cloud-chatops/README.md [hunk 1]
```diff
@@ -31,103 +30,4 @@ Events are defined in the `main.py/schedule_jobs` function:<br>
...
-}
-```
-Slack:<br>
-- TODO: How to create your own Slack application.<br>
-
-GitHub:<br>
--  A GitHub Personal Access Token is needed to bypass rate limiting and allows access to private repositories.<br>
-- Documentation on how to create a GitHub personal access token can be found 
-[here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).<br>
+For instructions on how to install and deploy Cloud ChatOps, see [INSTALL.md](./INSTALL.md)
```
Comment (khalford) [inline cloud-chatops/README.md:33]:
```suggestion
For instructions on how to deploy Cloud ChatOps, see [INSTALL.md](./INSTALL.md)
```

File: cloud-chatops/INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# Deploying a Cloud ChatOps application
```
Comment (khalford) [inline cloud-chatops/INSTALL.md:1]:
```suggestion
# Deploying Cloud ChatOps

File: cloud-chatops/INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+This document details the instructions to install and run ChatOps
```
Comment (khalford) [inline cloud-chatops/INSTALL.md:2]:
```suggestion
This document details the instructions to install dependencies and run the ChatOps application
```

**Review by khalford — CHANGES_REQUESTED** [review]
LGTM - just readability changes

File: cloud-chatops/README.md (hunk 1 above)
Comment (Neonbluestoplight) [reply cloud-chatops/README.md]: Thank you, have changed

**Review by khalford — APPROVED** [review]

**Pull request merged**